.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,71 @@
+name: Feature request
+description: Request a new feature to be added or improved in Godot Tools VS Code add-on
+labels:
+  - enhancement
+body:
+
+- type: markdown
+  attributes:
+    value: |
+      - Write a descriptive issue title above.
+      - Search [open](https://github.com/godotengine/godot-vscode-plugin/issues) and [closed](https://github.com/godotengine/godot-vscode-plugin/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported.
+      - Verify that you are using a [supported Godot version](https://docs.godotengine.org/en/stable/about/release_policy.html).
+      - This repository only contains the LSP *client*. Issues pertaining to the Godot editor itself or the LSP's *server* side should be reported on the [main Godot repository](https://github.com/godotengine/godot/issues).
+
+- type: input
+  attributes:
+    label: Godot version
+    description: >
+      Specify the Git commit hash if using a development or non-official build.
+      If you use a custom build, please test if your issue is reproducible in official builds too.
+    placeholder: 3.3.stable, 4.0.dev (3041becc6)
+  validations:
+    required: true
+
+- type: input
+  attributes:
+    label: VS Code version
+    description: >
+      Use the **Help > About** menu to see your current version.
+      Specify the Git commit hash if using a development or non-official build.
+      If you use a custom build, please test if your issue is reproducible in official builds too.
+    placeholder: "1.64.2"
+  validations:
+    required: true
+
+- type: input
+  attributes:
+    label: Godot Tools VS Code extension version
+    description: >
+      Open the **Extensions** side panel and click on the **godot-tools** extension to see your current version.
+      Specify the Git commit hash if using a development or non-official build.
+      If you use a custom build, please test if your issue is reproducible in official builds too.
+    placeholder: "1.1.3"
+  validations:
+    required: true
+
+- type: input
+  attributes:
+    label: System information
+    description: |
+      Specify the OS version, and when relevant hardware information.
+    placeholder: Windows 10
+  validations:
+    required: true
```

Comment:
I don't see how the used versions are relevant for a feature request?

Reply:
The used versions can be useful to know the *context* in which the feature is being requested. I've seen numerous feature requests made by users on old versions, despite the feature being implemented in a newer version :slightly_smiling_face:

Reply:
I know what you mean. But the usual solution I'm aware of is putting a warning text at the top - something like "_Please make sure that this feature wasn't already requested or already implemented in the main branch._" With your template a user could still create a feature request and simply enter an outdated version. To me this form just feels like someone forgot to edit it after copy-pasting the bug report template. It's even talking about "reproducing an issue".